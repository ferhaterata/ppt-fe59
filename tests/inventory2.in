AWF  5   00.45
