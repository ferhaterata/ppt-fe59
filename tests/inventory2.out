AWF 5 0.45
